publisher/htmlElementsCollector_test.go

```diff
@@ -129,29 +129,96 @@ func TestClassCollector(t *testing.T) {
 	}
 }
 
-func BenchmarkClassCollectorWriter(b *testing.B) {
```

Comment:
The primary motivation behind these benchmarks is to be able to compare before/after (using benchstat); but to do that, we cannot have a branch totally changing the existing benchmarks. I'm fine with adding more benchmarks, but we need to keep the old (at least until the relevant change is merged).

Reply:
I readded the old bench test, but left the new additions.